RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,67 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, and where we’d like to grow in the future. 
+
+
+## Core Strategy Statement  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## Who we are
+
+We are a community of technical folk. Writers, developers, editors and more. We are passionate about technical writing, documentation and open source.
```

Comment:
I'm reading this as the collective noun version of "folk," but that's because I'm familiar with that usage. And given there's a collective indicator already "community of," we might want to go with "folks" for the individual plurality. ("We are technical folk" would be a usage where the collective noun variant wouldn't be challenged, as an example of difference I'm seeing.)

And I was totally going to commit with the same nitpick, partly because it's serial in one statement and non in another.

Reply:
Fixed all the Oxford commas - so embarrassing!

Reply:
Removing folk from here as per @camerons earlier suggstion.